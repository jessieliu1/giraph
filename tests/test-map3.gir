int main() {
	graph<int> g = [A:1 -- B:3 -- C:5];
	map<node<int>> m;

	for_node(n : g) {
		m.put(n, n);
	}
	for_node(n : g) {
		print(n.data());
		print(m.get(n).data());
	}

	map<node<int>> m2;
	for_node(n : g) {
		m2.put(n, A);
	}
	for_node(n : g) {
		print(n.data());
		print(m2.get(n).data());
	}

	return 0;
}