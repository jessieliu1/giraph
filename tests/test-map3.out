1
1
3
3
5
5
1
1
3
1
5
1
